clang-tidy 16 fails in CI

### Describe the bug

clang-tidy 16 fails [in every CI run](https://github.com/CleverRaven/Cataclysm-DDA/actions/workflows/clang-tidy.yml) because FindSDL2_mixer.cmake can't find the flac dependency. This is probably caused by #68300 as it modified the tooling and tests started failing after it was merged.

Relevant error from an example [full example here](https://github.com/CleverRaven/Cataclysm-DDA/actions/runs/6302936294/job/17111136904) 
```
-- Found SDL2_mixer: /usr/lib/x86_64-linux-gnu/libSDL2_mixer.so  
-- Searching for SDL_mixer deps libraries --
-- Checking for module 'flac'
--   No package 'flac' found
CMake Error at /usr/local/share/cmake-3.27/Modules/FindPkgConfig.cmake:607 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/local/share/cmake-3.27/Modules/FindPkgConfig.cmake:829 (_pkg_check_modules_internal)
  CMakeModules/Find/FindSDL2_mixer.cmake:115 (pkg_check_modules)
  CMakeLists.txt:334 (find_package)


-- Configuring incomplete, errors occurred!
Error: Process completed with exit code 1.
```

### Attach save file

N/A

### Steps to reproduce

Have a commit in a PR that triggers clang-tidy 16 run

### Expected behavior

clang-tidy 16 setup to run successfully

### Screenshots

_No response_

### Versions and configuration

N/A

### Additional context

_No response_

--- a/.github/workflows/clang-tidy.yml
+++ b/.github/workflows/clang-tidy.yml
@@ -66,7 +66,7 @@ jobs:
           sudo apt-add-repository "deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-16 main"
           sudo apt update
           sudo apt install llvm-16 llvm-16-dev llvm-16-tools clang-16 clang-tidy-16 clang-tools-16 \
-            libclang-16-dev libsdl2-dev libsdl2-ttf-dev libsdl2-image-dev libsdl2-mixer-dev \
+            libclang-16-dev libflac-dev libsdl2-dev libsdl2-ttf-dev libsdl2-image-dev libsdl2-mixer-dev \
             libpulse-dev ccache gettext jq
     - name: install dependencies
       run: |

